tests/test_reorder_offsets.c

```diff
@@ -0,0 +1,106 @@
+/*
+  Copyright (C) 2019  Francesc Alted
```

Comment:
Copyright should be "The Blosc Developers" or "The Blosc Development Team".  Check with the LICENSE file.

Reply:
I changed it. Ok now?